opflexagent/vpplib/hook.py

```diff
@@ -117,8 +117,8 @@ def poll_vpp(self):
                 s = signaldict[abs(self.testcase.vpp.returncode)]
             else:
                 s = "unknown"
-            msg = "VPP subprocess died unexpectedly with returncode %d [%s]" % \
-                  (self.testcase.vpp.returncode, s)
+            msg = "VPP subprocess died unexpectedly with returncode %d [%s]" \
```

Comment:
Small nit: although not introduced with this commit, the use of backslashes to continue lines is discouraged. The convention is to use parenthesis if a statement needs to continue across more than one line.